picard/ui/coverartbox.py

```diff
@@ -378,14 +378,23 @@ def load_remote_image(self, url, mime, data):
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
+
+        if config.setting["behaviour_on_image_drop"] == 'replace':
```

Comment:
It's not neccesarily "remote", local files can also be dropped from any file manager. Maybe ```load_image_behavior``` ?